github/commitstatus_test.go

```diff
@@ -177,7 +177,7 @@ OAuth: X-Accepted-Oauth-Scopes: [], X-Oauth-Scopes: []`,
 			name: "non existing repo: Not Found",
 			repo: "non-existing-really",
 			wantErr: `Failed to add state "success" for commit 32e4b4f: 404 Not Found
-Body: {"message":"Not Found","documentation_url":"https://docs.github.com/rest/reference/repos#create-a-commit-status"}
+Body: {"message":"Not Found","documentation_url":"https://docs.github.com/rest/commits/statuses#create-a-commit-status"}
```

Comment:
Example of flaky test because too tight and dependent on upstream.
Maybe we should do a partial string match instead...